src/encoded/audit/file.py

```diff
@@ -73,7 +73,7 @@ def audit_file_controlled_by(value, system):
     if value['dataset'].get('assay_term_name') not in ['ChIP-seq', 'RAMPAGE', 'CAGE']:
         return
 
-    if 'target' in value['dataset'] and value['dataset']['target'].get('investigated_as') == 'Control':
+    if 'target' in value['dataset'] and 'control' in value['dataset']['target'].get('investigated_as'):
```

Comment:
This should say ``.get('investigated_as', [])`` as the default of get is ``None`` and ``'control' in None`` is an error.